src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
+
+    public void drawFigure() {
+        System.out.println("Figure color: " + color);
+    }
```

Comment:
```suggestion
    public void drawFigure() {
        System.out.println("Figure color: " + color);
    }
```

    move method draw to the separate interface Drawable, like AreaCalulator with the same logic for implementing

Reply:
Ok